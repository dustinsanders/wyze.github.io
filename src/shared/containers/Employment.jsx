// @flow

import type { Job, Jobs, VNode } from '../../types'
import Box from '../components/Box'
import Employer from '../components/Employer'
import preact from 'preact'

type Props = {
  jobs: Jobs,
}

const Employment = ({ jobs }: Props): VNode => (
  <Box wrap title="Employment">
    {jobs.map(( job: Job ): VNode => (
      <Employer key={job.name} {...job} />
    ))}
  </Box>
)

export default Employment
