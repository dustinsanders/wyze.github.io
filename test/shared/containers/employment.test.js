// @flow

import Employment from '../../../src/shared/containers/Employment'
import snapshot from '../../helpers/snapshot'

const jobs = [
  { name: 'Startup', start: 'Today' },
]

describe('<Employment />', () => {
  it('renders', snapshot.bind(null, Employment, { jobs }))
})
