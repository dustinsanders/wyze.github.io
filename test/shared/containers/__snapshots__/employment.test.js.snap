// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Employment /> renders 1`] = `
"<Box wrap title=\\"Employment\\">
  <Employer name=\\"Startup\\" start=\\"Today\\"></Employer>
</Box>"
`;
